kubeflow/asm/Makefile

```diff
@@ -43,7 +43,7 @@ install-asm: download-asmcli download-asm-package
 	--project_id $(PROJECT) \
 	--cluster_name $(NAME) \
 	--cluster_location $(LOCATION) \
-	--output_dir $(UPSTREAM) \
+	--output_dir ./package \
```

Comment:
Should we remove `UPSTREAM` variable in line 24?

Reply:
Fixed. Thank you @zijianjoy !